src/FTPServer.jl

```diff
@@ -84,14 +84,22 @@ function Server(
 
     cmd = `$PYTHON_CMD $SCRIPT $username $password $homedir --permissions $permissions`
     if security != :none
-        cmd = `$cmd --tls $security --cert-file $CERT --key-file $KEY --gen-certs TRUE`
+        cmd = `$cmd --tls $security --cert-file $CERT --key-file $KEY --gen-certs-dir $ROOT`
```

Comment:
`CERT` and `KEY` are currently full paths instead of just file names

Reply:
yeah, the python code uses realpath so it does end up working. I'm unsure if I should modify the python code to *only* do what we need it to do. Currently it has a few arguments we don't use, and stuff like that. Not sure it really needs to be "generic" so to speak since it's being used specifically for this package.

Reply:
If I modify it to just do what we need it to do then I can fix up the arguments and whatnot so they potentially make more sense.

Reply:
I suppose we can leave this as is and just open an issue. I think in the long term probably just allowing `--cert-file` to take a full path and not supplying a `--gen-certs-dir` would be reasonable. That would mean we'd need a flag to trigger the creation of the certs (by default it would just reference pre-created certs)

Reply:
Well, if I substantially change the python script, I don't think that would be breaking in any way, so I could change it up. Unless we just don't want to have too many changes in this one PR.